src/core/geocms/qgsgeonodeconnection.h

```diff
@@ -0,0 +1,62 @@
+/***************************************************************************
+    qgsgeonodeconnection.h
+    ---------------------
+    begin                : Feb 2017
+    copyright            : (C) 2017 by Muhammad Yarjuna Rohmat, Ismail Sunni
+    email                : rohmat at kartoza dot com, ismail at kartoza dot com
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGSGEONODECONNECTION_H
+#define QGSGEONODECONNECTION_H
+
+#define SIP_NO_FILE
+
+#include "qgis_core.h"
+#include "qgsdatasourceuri.h"
+#include "qgsmaplayer.h"
+#include "qgsproject.h"
+#include "qgsgeocmsconnection.h"
+
+#include <QNetworkReply>
```

Comment:
Most of the #includes here are not necessary... lowering the amount of #includes makes the builds faster...